Making Claude Cowork actually useful blog post (#16582)
PostHog/posthog.com: contents/blog/making-claude-cowork-actually-useful.md

---

> date: "2025-04-27"

Reviewer: we can set this to the right date just to check in the preview
```suggestion
date: "2025-04-24"
```

title: Making Claude Cowork actually useful
author:
  - charles-cook
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/robocop_8939115965.png
featuredImageType: full
category: Blog
tags:
- General
- Startups
- People
---

> ## Making Claude Cowork actually useful

Reviewer: ```suggestion
```

Every article I read about "AI agents for work" tells me the same things. Clean up my inbox. Summarize my meetings. Draft my emails. Turn rough notes into polished decks. Keep my calendar tidy.

> These are all lovely ideas. They are also, almost entirely, a list of low-leverage work that I already don't need help with or don’t want to do even better. 

Reviewer: ```suggestion
These are all lovely ideas. They are also, almost entirely, a list of low-leverage work that I already don't need help with or don’t want to do better. 
```

> I've been using Claude Cowork for a while now. Here's what I've actually found useful, what I haven't, and the single most useful thing it did. Which was telling me what to use it for (wait that bit was supposed to go at the end, oh well.)

Reviewer: ```suggestion
I've been using Claude Cowork for a while now. Here's what I've actually found useful, what I haven't, and the single most useful thing it did: telling me what to use it for (wait that bit was supposed to go at the end, oh well.)
```

**tl;dr**

1. Generic "top AI use cases" listicles are written for someone whose job is not your job. Don't copy them.  
2. The highest-leverage thing you can do is ask the agent to come up with use cases for *your* work.  
3. You need to give Cowork a set of "what the agent needs to know about me, my team, my company" files (which it keeps fresh on a schedule), otherwise everything else won’t work.  
4. I find the scheduled background tasks a lot more valuable than the on-demand chat.  
5. Cowork is still bad at some obvious things (most writing, navigating browsers, Google Drive).  
6. I'm a rational optimist about this, and you should be too.

I’ve also shared some of my scheduled tasks, in full, at the end of this post - copy and adapt them!

## Most AI agent advice is written for someone else's job

> When I started with Cowork I did the dutiful thing and read the "top use cases" posts. The advice was always a variation on the same list: inbox triage, meeting notes, polished drafts, calendar wrangling, create a PRD etc.

Reviewer: ```suggestion
When I started with Cowork, I did the dutiful thing and read the "top use cases" posts. The advice was always a variation on the same list: inbox triage, meeting notes, polished drafts, calendar wrangling, create a PRD, etc.
```

So "help me reply to emails" turns out to be the equivalent of asking Cowork to help me pour coffee. Possible! But not *obviously* the highest-leverage deployment of a reasoning engine in the universe.

This is probably why I have historically sucked at the productivity-hack genre and have used Asana like a noob for 12 years. 

## Ask the robot what to use it for

The single most useful hour I spent with Cowork was asking it to help me figure out what to use Cowork for.

I started by complaining about the things I don’t want to use it for. I then gave it a bunch of context about me (more later) and what I’m *already* doing in a typical week, rather than what aspirationally I think I should be doing. The suggestions were much better than anything I'd have come up with on my own. 

Two of my favourite scheduled tasks came directly out of that conversation.

### Coherence check

> Every Monday at 08:08 London time, Cowork pulls the quarterly objectives for every Small Team at PostHog (about fifty of them) from the website repo on GitHub. Then it reads them all in one pass, looking for the stuff you only spot when you actually have every plan in your head at the same time: two teams using the word "activation" to mean different things, Team A's plan assuming Team B will ship something that Team B has not committed to, two teams pulling in opposite directions on the same metric, objectives still labelled Q1 in late April. It posts a list to a private Slack channel, grouped by the person who owns each team. If nothing's worth flagging, it doesn't post at all. This is the kind of cross-org program management that you basically pay executives fabulous amounts of money to do.

Reviewer: ```suggestion
### 1. Coherence check

Every Monday at 08:08 London time, Cowork pulls the quarterly objectives for every [small team](/newsletter/small-teams) (about fifty of them) from the website repo on GitHub. Then it reads them all in one pass, looking for the stuff you only spot when you actually have every plan in your head at the same time: two teams using the word "activation" to mean different things, Team A's plan assuming Team B will ship something that Team B has not committed to, two teams pulling in opposite directions on the same metric, objectives still labelled Q1 in late April. 

It posts a list to a private Slack channel, grouped by the person who owns each team. If nothing's worth flagging, it doesn't post at all. This work is the kind of cross-org program management that you basically pay executives fabulous amounts of money to do.
```

### Push-harder Monday ritual.

> Also Monday, also before I open my laptop. Cowork pulls current PostHog metrics via our own MCP, reads the growth review sheets from the sales and marketing teams, and scans the last 7 days of the main GTM channels in Slack. Then it generates 3–5 aggressive growth bets that I might not have considered. Each bet has to name the “sacred cow” being sacrificed (CAC payback, margin, attribution clarity, brand consistency- pick one), rough size, a concrete first move with the person who'd own it, how we'd know to kill it, and a one-line rebuttal to the predictable frugal objection. The prompt literally says *"hype man, not McKinsey"* and *"most spicy first."* I don't take all of the bets seriously. I take about one in five, which is roughly the hit rate I want, adapting as needed. I then copy and paste my favorite to the relevant team and pretend I came up with it myself. 

Reviewer: ```suggestion
### 2. Push-harder Monday ritual

Also Monday, also before I open my laptop. Cowork pulls current PostHog metrics via [our own MCP](/docs/model-context-protocol), reads the growth review sheets from the sales and marketing teams, and scans the last 7 days of the main GTM channels in Slack. Then it generates 3–5 aggressive growth bets that I might not have considered. 

Each bet has to name the “sacred cow” being sacrificed (CAC payback, margin, attribution clarity, brand consistency- pick one), rough size, a concrete first move with the person who'd own it, how we'd know to kill it, and a one-line rebuttal to the predictable frugal objection. The prompt literally says *"hype man, not McKinsey"* and *"most spicy first."* 

I don't take all of the bets seriously. I take about one in five, which is roughly the hit rate I want, adapting as needed. I then copy and paste my favorite to the relevant team and pretend I came up with it myself. 
```

My setup lives under `~/Documents/Claude/Memory/`:

* **CLAUDE.md** is the index. Who I am, my direct reports (with GitHub handles and Slack IDs), current focus areas, standing preferences ("Meeting Free Tuesday — protect for deep work"), and a map pointing to everything else.  
* **glossary.md** explains stuff to Cowork. Every PostHog-specific acronym (NRR, TAM, MCP), every Slack channel that matters. When someone pings me "TAM coverage proposal status?", the agent knows TAM = Technical Account Manager at PostHog, not total addressable market. It also knows that when I refer to James, I’m probably talking about James Hawkins not James Greenhill (sorry Jams).   
* **people/[name].md** — one file per person I work closely with. What they own, how they want to be managed, what I've learned about working with them. This is the bit that helps Cowork help me prep for things like 1-1s.   

> * **context/company.md, context/handbook/, context/exec-responsibilities.md** — how the org actually works. Which exec owns which Small Teams. The current quarterly goals. How the handbook's structured and where to find what. (Side note - having a public handbook that consists entirely of .md files is awesome for this!)

Reviewer: ```suggestion
* `CLAUDE.md` is the index. Who I am, my direct reports (with GitHub handles and Slack IDs), current focus areas, standing preferences ("Meeting Free Tuesday — protect for deep work"), and a map pointing to everything else.  
* `glossary.md` explains stuff to Cowork. Every PostHog-specific acronym (NRR, TAM, MCP), every Slack channel that matters. When someone pings me "TAM coverage proposal status?", the agent knows TAM = Technical Account Manager at PostHog, not total addressable market. It also knows that when I refer to James, I’m probably talking about James Hawkins not James Greenhill (sorry Jams).   
* `people/[name].md` – one file per person I work closely with. What they own, how they want to be managed, what I've learned about working with them. This is the bit that helps Cowork help me prep for things like 1-1s.   
* `context/company.md`, `context/handbook/`, `context/exec-responsibilities.md` – how the org actually works. Which exec owns which small teams. The current quarterly goals. How the handbook's structured and where to find what. (Side note – having a [public handbook](/handbook) that consists entirely of `.md` files is awesome for this!)
```

To keep the context fresh, I have a bunch of scheduled tasks running, which all started with a prompt like ‘ok so how are you keeping this up to date’:

* **Daily memory sync.** Reconciles the working memory with whatever's changed in the underlying files.  
* **Weekly company-details sync.** Pulls the latest legal/finance/entity data from a Google Sheet into the memory file, so when I ask about the German entity I get current numbers, not something from February.  
* **Weekly handbook sync.** Grabs fresh versions of the handbook's root pages from the website repo.  

> * **Weekly Blitzscale ownership refresh.** Re-reads the handbook to work out which exec is currently responsible for which teams. This is what powers the "group by owner" bit of the coherence check - this matters because we change stuff around a bunch at PostHog.   

Reviewer: ```suggestion
* **Weekly Blitzscale ownership refresh.** Re-reads [the handbook](/handbook/exec/responsibilities) to work out which exec is currently responsible for which teams. This is what powers the "group by owner" bit of the coherence check - this matters because we change stuff around a bunch at PostHog.   
```

> * **Monthly memory refresh**. Every month I write a GitHub issue called "Things Charles cares about" that lists my current priorities. On the 8th, a task reads that issue and updates CLAUDE.md. The issue is the only place I write my monthly priorities down for the whole company to see.

Reviewer: ```suggestion
* **Monthly memory refresh**. Every month I write a GitHub issue called "Things Charles cares about" that lists my current priorities. On the 8th, a task reads that issue and updates `CLAUDE.md`. The issue is the only place I write my monthly priorities down for the whole company to see.
```

Later in the week:

* **Wednesday 08:00 - promise tracker.** Aka micromanagement on crack. It scans the past 7 days of my inbound Slack DMs, @-mentions, and replies, my Gmail, the transcripts of every Granola-recorded meeting I attended, and GitHub threads where I've asked a question. It looks for commitment language from other people ("I'll send you that by Friday," "leave it with me," "let me come back on this") and then cross-checks whether they delivered. I will probably have to delete it when the team tells me I’m being annoying.   
* **Friday 16:33 - weekly achievements report.** Pulls my calendar, my Granola transcripts, my sent Slack and Gmail, and my Asana tasks from Monday to Friday, and writes me a "your week in review" post: what I actually got done, where my time went, what I was talking about, and what's still in flight. Posted to Slack with a DM ping. This is mostly a morale thing, à la [Oliver Burkeman](https://www.oliverburkeman.com/donelist) (who would otherwise hate everything about this post, probably). 

## Where it still falls over

> Lots of things! But I’m 80% sure the models will just keep getting better and better. 

Reviewer: ```suggestion
Lots of things! But I’m 80% sure the models will just keep getting better.
```

  * I also believe that the implied social contract of ‘it should take the writer much more time to create something than the reader to read it’ will continue to hold.   
* **Browser actions.** If a task requires clicking around in a website, it's much slower and much more fragile than me doing it.  
* **Google Drive and Docs.** This is the one that actually frustrates me. Working with anything in Google Docs or Drive is a complete mess - the integrations are shallow and then you hit random edge cases like "the Google Drive integration can't actually move files between folders".   
* **Random integration gaps.** Related - you routinely run into something that *looks* like it should work ("just move this file", "just rename this doc") and then turns out to not be supported, with no obvious way to know that in advance.

I’m sure all of this stuff will get better. I’ve gone from ‘what’s an MCP’ to ‘wtf why does your MCP suck’ in the space of 6 months, so I imagine that companies just need to catch up. There are a couple of SaaS tools we use that don’t have good/any MCP support yet that I would love to plug into Cowork, but we’re not there yet. 

## Things can only get betterrr

> I'm excited about where this goes. The whole ‘ask the robot how to use it’ and then scheduling tasks taken together has taken me from messing about and creating slop to genuinely deriving value from AI tools. As PostHog grows, I’m already finding Cowork helps me keep on top of what’s going on without feeling like we need to impose more structure or project management to get things done. 

Reviewer: ```suggestion
I'm excited about where this goes. The whole ‘ask the robot how to use it’ and then scheduling tasks taken together has taken me from messing about and [creating slop](/blog/stop-ai-slop) to genuinely deriving value from AI tools. As PostHog grows, I’m already finding Cowork helps me keep on top of what’s going on without feeling like we need to impose more structure or project management to get things done. 
```

> I also think a lot of the early disappointment people have with agents is the "help me with my inbox" type stuff, where you’re getting the agent to do some low leverage task that is also hard to get right because the bar is "exactly how I would have done it, but faster." The bar is much easier to clear when the task is "do something I wasn't doing at all."

Reviewer: ```suggestion
I also think a lot of the early disappointment people have [with agents](/newsletter/building-ai-agents) is the "help me with my inbox" type stuff, where you’re getting the agent to do some low leverage task that is also hard to get right because the bar is "exactly how I would have done it, but faster." The bar is much easier to clear when the task is "do something I wasn't doing at all."
```

- Source link
- When made
- Stated deadline if any (resolve "by Friday" to a date)

## Step 3 — Check for delivery
Look for: subsequent follow-up with the deliverable attached, a new artefact
by the committer, my own acknowledgement ("thanks, got it"), or explicit
retraction.

Drop same-day items (no time to deliver) and future deadlines not yet passed.
When in doubt, leave it in.

## Step 4 — Group, dedupe, post
- Dedupe across sources (often the same promise appears in a meeting and a
  follow-up DM).
- Group by committer. Sort oldest-first within each group.
- Priority to the top: execs, external parties, anything aged >5 days.

Post to [channel]. Bullets, short date stamps ("made Mon", "5d old", "due Fri —
overdue 2d"). No emojis. If nothing material, do NOT post.
```
## Bonus: the MCPs I actually use

If the scheduled tasks are the output layer and the memory files are the context layer, the MCPs are the plumbing underneath that connects the agent to the rest of my stack. A few I'd actually recommend, in rough order of how often Cowork reaches for them on my behalf. Cowork calls these ‘Connectors’ to stop non-technical people like me from being scared. 

### PostHog MCP

> The one that changes what's possible. When I ask "how's ARR looking this week?" Cowork runs actual SQL against our PostHog data warehouse, pulls live numbers, and hands them back.

Reviewer: ```suggestion
The one that changes what's possible. When I ask "how's ARR looking this week?" Cowork runs actual SQL against [our data warehouse](/data-stack), pulls live numbers, and hands them back.
```

> I also use it for customer lookups. I route "who's the TAM for [customer]?" through the PostHog MCP because we keep Vitally synced into our warehouse. Every customer question Cowork answers starts with an `execute-sql` call against that data.

Reviewer: ```suggestion
I also use it for customer lookups. I route "who's the TAM for [customer]?" through the PostHog MCP because we keep [Vitally synced into our warehouse](/docs/cdp/sources/vitally). Every customer question Cowork answers starts with an `execute-sql` call against that data.
```
